WalletWasabi/WabiSabi/Client/Banning/CoinPrison.cs

```diff
@@ -78,6 +82,22 @@ private void ToFile()
 		File.WriteAllText(FilePath, json);
 	}
 
+	private DateTimeOffset ReduceBanningTimeIfNeeded(DateTimeOffset bannedUntil)
+	{
+		var currentDate = DateTimeOffset.UtcNow;
+		if (bannedUntil > currentDate.AddDays(_absoluteMaximumLocalPrisonBanningTimeInDays))
+		{
+			Random random = new();
+			int randomHours = random.Next(24, 49);
```

Comment:
you must use _absoluteMaximumLocalPrisonBanningTimeInDays * 24 - 1 as max and Math.Round((_absoluteMaximumLocalPrisonBanningTimeInDays * 24 - 1) / 2) as min